src/api/public/apidocs-new/paths/staging_project_name_staging_projects_staging_project_name.yaml

```
@@ -0,0 +1,89 @@
+get:
+  summary: Gets the overall state of a staging project.
+  description: |
+    Extra information can be requested by adding any combination of these parameters in the URL: requests, status and history.
+    - If requests is present, the output includes the staged, untracked and obsolete requests as well as missing reviews.
+    - If status is present, the output includes the overall state and the status xml (broken packages, missing reviews, checks, etc.)
+    - If history is present, the output includes the history of the staging project.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/staging_project_name.yaml'
+    - name: requests
+      in: query
+      schema:
+        type: string
+        enum:
+          - 1
+          - 0
+      default: 0
```

```suggestion
        example:
          1
```

Removing the `0` as an alternative to `1` is missing. But I think it will be easier if you rebase on master, as this file (without `0 `) is already there.